template/template_example.Rmd

```diff
@@ -0,0 +1,206 @@
+
+The `<experiment_accession_id>` folder has the data and metadata TSV files you will need for this example analysis.
+Experiment accession ids usually look something like `GSE1235` or `SRP12345`. 
+
+## Checkout our file structure!
+
+Your new analysis folder should contain: 
+
+- The example analysis Rmd you downloaded  
+- A folder called "data" which contains:  
+  - The gene expression  
+  - The metadata TSV  
+
+Your example analysis folder should now look something like this (except with respective experiment accession id and analysis notebook name you are using): 
+
+<img src="screenshots/analysis-folder-structure.png" width=400>
+
+In order for our example here to run without a hitch, we need these files to be in these locations so we've constructed a test to check before we get started with the analysis. 
+Run this chunk to double check that your files are in the right place. 
+
+```{r}
+# Check if the gene expression matrix file is in the data directory stored as `data_dir`
+file.exists(file.path(data_dir, <<DATA_ACCESSION FILENAME>>))
+
+# Check if the metadata file is in the data directory stored as `data_dir`
+file.exists(file.path(data_dir, <<METADATA_ACCESSION FILENAME>>))
+```
+
+If the chunk above printed out `FALSE` to either of those tests, you won't be able to run this analysis _as is_ until those files are in the appropriate place.
+
+If the concept of a "file path" is unfamiliar to you; we recommend taking a look at our [section about file paths](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#an-important-note-about-file-paths-and-Rmds). 
+
+# Using a different refine.bio dataset with this analysis?
+
+If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory you created and changing the filenames and paths in the notebook to match these files (we've put comments to signify where you would need to change the code).
+We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebook.
+From here you can customize this analysis example to fit your own scientific questions and preferences. 
+
+***
+
+# <<NAME OF example>>
+
+## Install libraries
+
+See our getting started with [package installation](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#what-you-need-to-install) for a list of the other software you will need, as well as more tips and resources. 
+
+In this analysis, we will be using <<PACKAGE>> [<<CITATION>>].
+
+```{r}
+if (!(<<PACKAGE>> %in% installed.packages())) {
+  # Install this package if it isn't installed yet
+  BiocManager::install(<<PACKAGE>>, update = FALSE)
+}
+```
+
+Attach the <<PACKAGE>>:
+
+```{r}
+# Attach the library
+library(<<PACKAGE>>)
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
```

Comment:
I have said elsewhere that I really don't like this, as I think it is confusing to people, with all the backticks and whatnot. I much prefer `library(magrittr)`, Even though it loads a few other things that people will never use, I have never had it conflict.

Reply:
Okay. Fine with me!